fiftyone/utils/image.py

```diff
@@ -74,6 +74,15 @@ def transform_images(
     The ``filepath`` of the samples are updated to point to the transformed
     images.
 
+    .. note::
+
+        This method will not update any outdated metadata fields on the collection
+        after resizing. This can cause annotations to show in the wrong place. To avoid
```

Comment:
Our style guide requests < 80 character line limit. Could you update the line breaks to satisfy that please? :)

Reply:
Sure thing.